src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -642,12 +745,53 @@ public void testLastIndexOfRandom() {
                 }
             }
             
-            final int result = Utils.lastIndexOf(reference, query);
+            final int result = Utils.lastIndexOfAtMostTwoMismatches(reference, query, 0);
             final int expected = new String(reference).lastIndexOf(new String(query));
             Assert.assertEquals(result, expected);
         }
     }
 
+    @Test
+    public void atMostOneIndel(){
```

Comment:
I would change this to a data provider that has the firlds reference, query and, expected result.